implementations/rust/ockam/ockam_command/src/relay/show.rs

```diff
@@ -33,23 +33,14 @@ async fn run_impl(ctx: Context, (opts, cmd): (CommandGlobalOpts, ShowCommand)) -
     let mut rpc = Rpc::background(&ctx, &opts, &node_name)?;
     rpc.request(Request::get(format!("/node/forwarder/{remote_address}")))
         .await?;
-    let forwarder_info_response = rpc.parse_response::<ForwarderInfo>()?;
+    let relay_info_response = rpc.parse_response::<ForwarderInfo>()?;
```

Comment:
I think we should go all the way down and also rename `ForwarderInfo`, `ockam::Forwarder` etc... in order to keep our vocabulary consistent across the whole stack

Reply:
I agree. I had two thoughts though.

 I didn't want to balloon this PR; so I just started with this one crate, `ockam_command`
 
 Also, wanted to be sure it was applicable moving all the way down the stack.